### DISCUSSION TITLE
How do I turn off tooltips at the end of a trend line?

### DISCUSSION THREAD

**Discussion by thadkrugman** [recharts/recharts#3356] [Q&A] [answered]
Hello,

I have a scatter chart with a trendline. The trendline starts on the first scatter cell and ends on the last one. 

Here is the problem, when I hover over the first or last scatter cell, I get a tooltip for the end of the line and not for the scatter cell. Is there a way to turn off the tooltips for the ends of the trendline?

**Comment by ckifer** [comment, marked as answer]
Hmm could you provide a codesandbox example?

**Reply by thadkrugman** [reply]
https://codesandbox.io/s/trusting-https-jrt0yz?file=/src/App.js

I got the sandbox working.

**Reply by ckifer** [reply]
Thanks, I'll take a look

**Reply by ckifer** [reply]
Yeah so in general for something like a trendline (especially if you don't want a tooltip) I'd probably just use a `ReferenceLine` - https://recharts.org/en-US/api/ReferenceLine

But in your case this is happening because your custom tooltip is custom and it doesn't take `tooltipType` into account.
Adding a check for tooltip type ` if (active && payload && payload.length && payload?.[0]?.type !== "none") {` to the CustomTooltip resolves the issue

**edit**: though that doesn't resolve the issue entirely cause now there is no tooltip at all - you'll have to check which payload key you're looking at when you render your tooltip content

**Reply by ckifer** [reply]
Try this for your Reference Line

```jsx
          <ReferenceLine
            ifOverflow="extendDomain"
            segment={[
              { x: trendFirstCoord.date, y: trendFirstCoord.score },
              { x: trendLastCoord.date, y: trendLastCoord.score }
            ]}
            stroke="#0cb2da"
            strokeDasharray={"6, 2"}
          />
```

and be sure to add `dataKey="score"` to your scatter - I think that was causing issues

**Reply by ckifer** [reply]
https://codesandbox.io/s/objective-lucy-38use1

**Reply by thadkrugman** [reply]
Thank you so much! That fixed the issue and it is working as intended now. I really appreciate your help.
